index.js

```diff
@@ -1,6 +1,11 @@
 'use strict';
+const {spawnSync} = require('child_process');
 const path = require('path');
-const execa = require('execa');
+const stripEof = require('strip-eof');
+
+const spawn = (cmd, args) => stripEof(spawnSync(cmd, args || [], {encoding: 'utf8'}).stdout);
```

Comment:
You don't actually need `stripEof`. You can just use `String#trim()`.

Reply:
Right, nice 👍